lighthouse-core/gather/driver.js

```diff
@@ -150,7 +150,11 @@ class Driver {
         returnByValue: true
       }).then(result => {
         clearTimeout(asyncTimeout);
-        resolve(result.result.value);
```

Comment:
this was pretty nasty. rejected promises returned from those functions executed in the browser aren't actually rejected here. instead it passes minimal information to `result.exceptionDetails`